modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/HandleActionClick.ts

```diff
@@ -0,0 +1,158 @@
+/**
```

Comment:
We should consider moving all actions helpers and utils into a `/utils` folder under `/actions`. Not need to do now.

I will wait for Marko to confirm  :smile: , but files should be named after its function name `handleActionClick` (as it is not a component function)